Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean

```diff
@@ -507,5 +507,3 @@ theorem mem_span_fractionalIdealLatticeBasis (x : (E K)) :
   rfl
 
 end integerLattice
-
-end NumberField.mixedEmbedding
```

Comment:
I know it's not necessary, strictly speaking, but I'd keep this line.

Reply:
Yes. you're right.